### DISCUSSION TITLE
Anime requests

### DISCUSSION THREAD

**Discussion by rocktimsaikia** [AnimechanOrg/animechan#65] [General]
If you don't find the anime you are looking for. Comment down below to start a thread. And It might get added to the upcoming database updates.

Initially, we were tracking new anime requests in issue #39 but we have recently moved to Discussion for better community engagement.

We are periodically adding new animes to the database picking from requests you guys leave. 

> Some recently added Animes:
- [x] _Attack on Titan_ 
- [x] _Jujutsu Kaisen_ 
- [x] _My Hero Academia_ 
- [x] _Demon Slayer_ 
- [x] _Tokyo Revengers_ 
- [x] _Dr Stone_ 
- [x] _Black Clover_ 
- [x] _Steins Gate_

**Comment by skouv05** [comment]
Fire force

**Comment by iArmanKarimi** [comment]
Chainsaw man

**Reply by faralost** [reply]
+++

**Comment by MCJUNIORLK** [comment]
Darling in the Franxx

**Comment by 123zxk** [comment]
您好，邮件已收到，会尽快答复。谢谢！

**Comment by anacgsantana** [comment]
One Piece

**Comment by sunny4260** [comment]
Death note

**Comment by DiOality** [comment]
hunter X hunter

**Comment by DiOality** [comment]
How do I see the entire list of what is available?

**Comment by BhupathiKMS** [comment]
api - "https://animechan.xyz/api/random" seems to be not working now. Please check

**Reply by BhupathiKMS** [reply]
Thanks for the immediate workaround. It is working now

**Comment by jay-babu** [comment]
classroom of the elite

**Comment by rocktimsaikia** [comment]
![image](https://github.com/user-attachments/assets/70fb4bff-bcaf-4773-ad35-ca8548459eed)
https://buymeacoffee.com/animechan_api/update-new-anime-quotes-added

**Comment by MwaniN** [comment]
I'm not sure if you have these already yet it would be cool to see quotes from:

- Akira
- Ghost in the Shell (1995)
- Ghost in the Shell:  Stand Alone Complex
- Samurai Champloo
- Fist of the North Star (1986)
- Delicious In Dungeon
- Frieren
- Excel Saga

Thanks!
